cmd/vclusterctl/cmd/platform/add/vcluster.go

```diff
@@ -33,7 +33,7 @@ vcluster platform add vcluster my-vcluster --namespace vcluster-my-vcluster --pr
 	`
 
 	addCmd := &cobra.Command{
-		Use:   "vcluster",
+		Use:   "vcluster VCLUSTER_NAME",
 		Short: "Adds an existing vCluster to the vCluster platform",
 		Long:  description,
 		Args:  cobra.ExactArgs(1),
```

Comment:
Also this here: https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/create.go#L46